external_reports/libgit2.yml

```diff
@@ -0,0 +1,297 @@
+---
+name: libgit2
```

Comment:
Should the zlib issues in libgit be here or in zlib?

Reply:
I don't think I know enough about that issue to have an answer. If the zlib is distribute with the module, even inside a folder called libgit2, I'd report it with zlib. If not, that means that there are some other sticky cases. Let me look at this more closely.